src/NixFromNpm/Npm/Types.hs

```diff
@@ -25,12 +25,33 @@ data PackageInfo = PackageInfo {
   piTags :: Record SemVer
   } deriving (Show, Eq)
 
+-- | Taken from https://nodejs.org/api/process.html#process_process_platform,
+-- and filtered to those that correspond to nixpkgs platforms.
+data NodePlatform
+  = DarwinPlatform
```

Comment:
The `Platform` suffix seems redundant to me.

Reply:
Just due to how namespacing works in Haskell. Since it's not likely to be used elsewhere and some of those are generic enough to be likely to conflict with names in other contexts.

Reply:
Decided it was OK to nuke it